src/mixins/borrow/cauldrons.js

```diff
@@ -506,7 +513,7 @@ export default {
       if (poolInterest) interest = poolInterest;
       if (!poolInterest && pool?.interest) interest = pool?.interest;
 
-      if(pool.id === 36 && this.chainId === 1) interest = pool.interest; // deprecated WBTC
+      if (pool.id === 36 && this.chainId === 1) interest = pool.interest; // deprecated WBTC
```

Comment:
This is weird. Do we still need this or did we get rid of it when we stopped supporting v1 cauldrons?